common/lib/xmodule/xmodule/js/src/sequence/edit.js

```diff
@@ -0,0 +1,19 @@
+/* globals XModule */
+/* eslint-disable no-underscore-dangle */
+
+(function() {
```

Comment:
It feels like overkill to add all of this for inheritance that is doing nothing. 

class @SequenceDescriptor extends XModule.Descriptor